src/main.tsx

```diff
@@ -29,6 +29,7 @@ const wagmiConfig = defaultWagmiConfig({ chains, projectId, metadata })
 
 createWeb3Modal({
   wagmiConfig,
+  enableAnalytics: true,
```

Comment:
This needs a prod env flag I think like here? https://github.com/WalletConnect/cloud-app/pull/703/files#diff-88a36bc7a53bfa58c6f451464798d631d785160dfacfdaf9479be7a0b1d7e5e5R47

We don't want to send events for dev activity.